src/actions/pkg.cr

```diff
@@ -1,3 +1,5 @@
+require "../actions"
```

Comment:
Not needed

Reply:
My editor hooks in with the compiler to alert me to issues with types or syntax while I'm working, and that feature is only effective if each file has the appropriate `require` statements. It doesn't hurt anything to be there.

Reply:
This is a non sense, this file is required by a file (`actions.cr`), that is also required back?!
I don't know about your editor, but this isn't a strong enough argument against this anti-patterns.

Nobody wants to see this unneeded backward parent requirements everywhere.

Reply:
:open_mouth: wow, if it's that important to you...

Reply:
Yes it is, I don't invent anything there, I'm looking at how other big projects like crystal, shards, lucky and other non-crystal projects.

That's a modularization principle: why here `pkg` will require `actions`, that will bring `src`, `app` etc?

Only import/require only what is needed – simpler to understand, avoid circle dependencies.